tests/sanity/sanity_test.go

```diff
@@ -53,6 +53,7 @@ var _ = AfterSuite(func() {
 })
 
 var _ = Describe("AWS FSx for Lustre CSI Driver", func() {
+	_ = os.MkdirAll("/tmp/csi", os.ModePerm)
```

Comment:
Why is this needed?

Reply:
sanity used to MkdirAll the paths for us, but now it only does Mkdir causing tests to fail without this (locally and in travis): https://github.com/kubernetes-csi/csi-test/commit/267dbbcb874cf12332ec966f372a10ad8df246d6#diff-cdf6b6b472a77c21dd6f192b6f3241eaL262